macros/REST_MakeMetadata.C

```diff
@@ -70,6 +70,8 @@ Int_t REST_MakeMetadata(TString name) {
     headerFile << "    /// REMOVE MEMBER. A dummy member that will be NOT written to the ROOT file." << endl;
     headerFile << "    Double_t fDummyVar = 3.14; //!" << endl;
     headerFile << endl;
+    headerFile << "    void Initialize();" << endl;
```

Comment:
Don't quite understand this `Initialize`, there is another `public` one below properly tagged as `override`

Reply:
I just moved the method from public to private for some reason I dont recall now.

Reply:
Probably is the conflict resolution that kept both methods. Now I just added the override at the private and removed the public.